cmd/dlv/cmds/commands.go

```diff
@@ -190,6 +190,23 @@ execution is resumed at the start of the debug session.`,
 	}
 	rootCommand.AddCommand(dapCommand)
 
+	// 'dap-reverse' subcommand - this is for internal use only.
```

Comment:
IMO this should just be a flag for the DAP command:

```
dlv dap --connect=127.0.0.1:13370
```

Reply:
Done.
I was originally reluctant to let a flag change `dap` (it's not a server any more) and wanted to hide this from users completely.
Still marked the flag hidden - the cobra doc generator inserts an empty "Options" block. I will see if updating the cobra dependency would fix this documentation issue.